scripts/events_module/generate_events.py

```diff
@@ -222,16 +230,16 @@ def possible_short_events(event_type=None):
 
         return event_list
 
-    @staticmethod
```

Comment:
Why was this changed from a static method?

Reply:
Good question, i have no idea xD I'll put that back